plugins/destiny.py

```diff
@@ -566,9 +562,13 @@ def lore(text, bot, notice):
 def collection(text, nick, bot):
     text = nick if not text else text
     membership = get_user(text)
+    links = CACHE['links'].get(nick)
```

Comment:
This might need a default value if for some reason the user isn't correctly linked? But I think that error handling is already in get_user.